frontend/src/app/components/Header.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import { Link } from 'react-router';
-import classnames from 'classnames';
 
 import LayoutWrapper from './LayoutWrapper';
 import RetireConfirmationDialog from './RetireConfirmationDialog';
+import Settings from './Settings';
 
 export default class Header extends React.Component {
 
   constructor(props) {
     super(props);
+
     this.closeTimer = null;
+    this.showSettingsMenu = this.showSettingsMenu.bind(this);
```

Comment:
I prefer to put the call to bind at the place where this.showSettingsMenu is being passed, but I know I am probably in the minority about that.

It just seems gross to me to have some functions on this that are bound, and some that are not.